Documentation/contributing/development/images.rst

```diff
@@ -124,95 +124,72 @@ Image dependency:
 Update cilium-builder and cilium-runtime images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+This section will require you to install `Skopeo <https://github.com/containers/skopeo>`__.
+
 Login to quay.io with your credentials to the repository that you want to
 update:
 
-`cilium-builder <https://quay.io/repository/cilium/cilium-builder?tab=builds>`__ - contains Cilium build-time dependencies
-`cilium-runtime <https://quay.io/repository/cilium/cilium-runtime?tab=builds>`__ - contains Cilium run-time dependencies
-
-0. After login, select the tab "builds" on the left menu.
-
-.. image:: ../../images/cilium-quayio-tag-0.png
-    :align: center
+`cilium-runtime <https://hub.docker.com/repository/docker/cilium/cilium-runtime-dev/builds>`__ - contains Cilium run-time dependencies
 
-1. Click on the wheel.
-2. Enable the trigger for that build trigger.
+#. After login, click the "Configure Automated Builds" button.
 
-.. image:: ../../images/cilium-quayio-tag-1.png
-    :align: center
+#. Add a new build rule, taking care to point to the correct Dockerfile and
+   Build context, and tagging with the date and suffix ``-v1.9``.
 
-3. Confirm that you want to enable the trigger.
+   .. image:: ../../images/cilium-dockerhub-tag-0.png
+       :align: center
 
-.. image:: ../../images/cilium-quayio-tag-2.png
-    :align: center
+#. Delete all of the old build rules.
 
-4. After enabling the trigger, click again on the wheel.
-5. And click on "Run Trigger Now".
+#. Click "Save and Build".
 
-.. image:: ../../images/cilium-quayio-tag-3.png
-    :align: center
+#. Wait for DockerHub to build the image.
 
-6. A new pop-up will appear to select your desired branch.
-7. If you're interested in simply bumping the image to have the latest
-   packages, then select the release branch (i.e. v1.7, v1.8). If you already
-   have a branch that contains changes, select the branch that contains the new
-   changes.
+#. Log into skopeo into both dockerhub and quay.io, and copy the image across:
 
-.. image:: ../../images/cilium-quayio-tag-4.png
-    :align: center
+   .. code-block:: shell-session
 
-8. After selecting your branch click on "Start Build".
+      $ skopeo login docker.io
```

Comment:
Does it ask prompt for a username?

Reply:
I think so, though I don't know what kind of configuration I may have prepared locally to make this work. I'd imagine that if someone other than Andre / I need to follow these instructions, then hopefully this step will at least ensure that they figure out how to set up skopeo correctly.

That said, this is only applicable for v1.9 as the newer branches have better infrastructure in the tree for handling these updates so we don't need to rely on developers having direct access to the upstream container repositories.